telemetry/specs/INT.mdk

```diff
@@ -397,17 +404,20 @@ receiver to ignore the value of the Checksum field. For UDP over IPv6, there
 are specific use cases in which it is possible to assign a zero Checksum 
 (as defined in RFC 6936).
 
-If an L4 Checksum update is required, an INT source/transit switch may perform
+If an L4 Checksum update is required, an INT source/transit/sink switch may perform
```

Comment:
This does not work. The sink cannot use checksum complement, nor can it assume that removal of checksum complement fields is enough. If an L4 checksum update at the sink is required, it must update the L4 Checksum field.